system-test/system-test-script.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#Stopping the script if any command fails
+set -euo pipefail
+export myvar="system-test/logfile"
+cd ..
+ echo "Starting collection roller....."
+bin/collection-roller 2>&1 > $myvar &
+#Getting the process id of collection roller
+pid1=$!
+#Checking if the collection roller service has started
+if [ -z "$pid1" ]
+then
+      echo "Collection Roller is not running"
+else
+      echo "Collection Roller is running....."
+fi
+echo "Starting alert engine....."
+bin/alert-engine 2>&1 > $myvar &
+#Getting process of alert engine
+pid2=$!
+#Checking if the alert engine service has started
+if [ -z "$pid2" ]
+then
+      echo "Alert Engine is not running"
+else
+      echo "Alert Engine is running...."
+fi
+echo "Starting log collector...."
+bin/log-collector 2>&1 > $myvar &
+#Getting process of log collector
+pid3=$!
+#Checking if the log collector service has started
+if [ -z "$pid3" ]
+then
+      echo "Log collector is not running"
+else
+      echo "Log Collector is running...."
+fi
+echo "Collection roller PID: " $pid1
+echo "Log Collector PID:" $pid3
+echo "Alert Engine PID:" $pid2
```

Comment:
Are you actually killing the process somewhere?

Reply:
yes